websites/A/AniWorld/presence.ts

```diff
@@ -194,6 +193,62 @@ async function getStaticPages(): Promise<{ [key: string]: StaticPageInfo }> {
       smallImageKey: Assets.Writing,
       smallImageText: strings.editinfo,
     },
+    '/user/profil': {
+      details: strings.profile,
+      smallImageKey: Assets.Reading,
+      smallImageText: strings.profile,
+    },
+  }
+}
+
+let lastAnimeKey: string | null = null
+let cachedAnimeData: Awaited<ReturnType<AnimeDataFetcher['loadAnimeData']>> | null = null
+let isLoadingAnimeData = false
+let lastLoadPromise: Promise<Awaited<ReturnType<AnimeDataFetcher['loadAnimeData']>>> | null = null
+
+function getAnimeKeyFromUrl(url: string): string | null {
+  const match = url.match(/\/anime\/stream\/(.+?)\/?$/);
+  return match && typeof match[1] === 'string' ? match[1] : null;
+}
+
+async function getCachedAnimeData(): Promise<AnimeDataFetcher['animeData'] | null> {
+  const key = getAnimeKeyFromUrl(window.location.pathname)
```

Comment:
For consistency, use `document.location.pathname` instead of `window.location.pathname`